io/StacReader.hpp

```diff
@@ -31,68 +31,63 @@
 * OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY
 * OF SUCH DAMAGE.
 ****************************************************************************/
+
 #pragma once
 
 #include <time.h>
 #include <ctime>
 
 #include <pdal/PointView.hpp>
 #include <pdal/Reader.hpp>
-
-
 #include <pdal/JsonFwd.hpp>
 #include <pdal/StageFactory.hpp>
 #include <filters/MergeFilter.hpp>
-
+#include "private/stac/Item.hpp"
```

Comment:
Shouldn't be including this from a public header.  I think you're just using it for the `Item` class but its only usage here is by reference, so it can be forward-declared.